### PR TITLE
Import changes from release not yet on master

### PR DESCRIPTION
**Pull request by TheoPannetier**

### CODE REVIEW

File: src/Common/BatchMode.cpp [hunk 1]
```diff
@@ -5312,761 +3496,65 @@ bInitIndsFile >> year;
...
-	bInitIndsFile >> year;
-	if (bInitIndsFile.eof()) year = -98765;
-} // end of while loop
-if (!bInitIndsFile.eof()) {
-	EOFerror(filetype);
-	errors++;
-}
-
-return errors;
-}
```
Comment (TheoPannetier) [inline src/Common/BatchMode.cpp:5372]: Shouldn't this part stay in the base version?

Comment (TheoPannetier) [reply src/Common/BatchMode.cpp:5372]: No, git is getting a bit confused by RS_ABC but this bit is re-added below

**Comment by JetteReeg** [comment]
Hi Theo, I am afraid that you haven't been working with the git subtree feature? So now all changes you have done on Friday are only applied on the RangeShifter_resturct repository, but not on the RScore repository. The idea was, that we use the git subtree feature for the RScore subdirectory and push changes to the RScore repository (using `git subtree push`) as well as to the RangeShifter_restruct (or RangeShiftR_restruct) repository. In that way both the batch version as well as the R package can access changes/commits/etc. To setup the git subtree, you need to checkout the branch, delete the RScore subdirectory and add it again as a subtree (which will checkout the given branch of the RScore repository into the RScore subdirectory. I have provided instructions in the word document (and I thought also in the ppt?). I know that this step might by a bit annoying, but I haven't found a better solution, so both batch and R package would link to the same RScore files/repository. (But if you know a better solution, I am happy to discuss it.)
Best, Jette

**Comment by TheoPannetier** [comment]
@JetteReeg ah shoot, I had indeed completely overlooked the subtree feature, sorry! Let me learn a bit more about it and I'll be setting it up shortly

**Pull request merged**